### PR TITLE
Add primary option to bond creation master

### PR DESCRIPTION
**Pull request by MosheSheena**

### CODE REVIEW

**Comment by MosheSheena** [comment]
#133 is blocking the CI for this PR

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ sudo: required
 services:
   - docker
 python:
-- '3.5'
 - '3.6'
+- '3.7'
```
Comment (myakove) [inline .travis.yml:7]: Didn't you say that you split this PR?

**Review by myakove — COMMENTED** [review]
Otherwise LGTM

File: .travis.yml (hunk 1 above)
Comment (MosheSheena) [reply .travis.yml:7]: I did. what you commented about is from PR #133.
I just rebased this branch over newly updated master, and somehow it references it here.

Comment (MosheSheena) [reply .travis.yml:7]: This branch was intended to have 2 commits only:
 - 64842cf
 - 1f7f44f

**Comment by MosheSheena** [comment]
Hi, @myakove do you think this PR can be merged?

**Comment by myakove** [comment]
LGTM

**Comment by lukas-bednar** [comment]
@MosheSheena please just rebase your change .

I tried it locally on my laptop and I was successful to rebase on rhevm-qe-automation/master. And I see there should be only two commits in this PR.

Please try again.

**Review by lukas-bednar — APPROVED** [review]

**Pull request merged**